Gemfile

```diff
@@ -6,4 +6,4 @@ gemspec
 rails_version = "#{ENV['RAILS_VERSION'] || 'main'}"
 
 gem "capybara", "~> 3"
-gem "rails", rails_version == "main" ? { git: "https://github.com/rails/rails", ref: "main" } : rails_version
+gem "rails", "6.1.3.2"
```

Comment:
We do not want version `7.0.0.alpha` that needs ruby version `2.7.0` in which the bug does not appear anymore... Downgrade Rails so we can validate that test fail before the fix with a previous version.

⚠️ b4bbbaee19dc6012ac543b367243363f673ba4c2 should be revert before merge!